docs/guides/rendering.md

```diff
@@ -1,445 +1,350 @@
-canvas worth of data, and you cannot interrupt the load to further adjust
-the camera.
-
-With `NAPARI_ASYNC` overall performance is the same, but the advantage is
-you can interrupt the load by moving the camera at any time. This is a nice
-improvement, but working with slow-loading data is still slow. Most large
-image viewers improve on this experience with chunks or tiles. With chunks
-or tiles when the image is panned the existing tiles are translated and
-re-used. Then the viewer only needs to fetch tiles which newly slid onto
-the screen. This style of rendering is what the `NAPARI_OCTREE` flag
-enables.
-
-## NAPARI_OCTREE
-
-Set `NAPARI_OCTREE=1` to use the experimental
-{class}`~napari.layers.image.experimental.octree_image.OctreeImage` class
-instead of the normal {class}`~napari.layers.Image` class. The
-new {class}`~napari.layers.image.experimental.octree_image.OctreeImage`
-class will use the same
-{class}`~napari.components.experimental.chunk._loader.ChunkLoader` that
-`NAPARI_ASYNC` enables. In addition, `NAPARI_OCTREE` will use the new
-{class}`~napari._vispy.experimental.tiled_image_visual.TiledImageVisual`
-instead of the Vispy `ImageVisual` class that napari's
-{class}`~napari.layers.Image` class uses.
-
-```{note}
-The current `OCTREE` implementation only fully supports a single 2D image and
-may not function with 3D images or multiple images. Improving support
-for 3D and multiple images is part of future work on the `OCTREE`.
+# Rendering
+
+This document explains how napari produces a 2- or 3-dimensional render in the canvas from layers' n-dimensional array-like data.
+The intended audience is someone who wants to understand napari's rendering pipeline to help optimize its performance for their usage.
+Or someone that wants to contribute to and help improve the clarity or performance of napari's rendering pipeline itself.
+
+## Overview
+
+At a high level, rendering in napari is simple.
+
+1. Viewing: {attr}`ViewerModel.dims<napari.components.ViewerModel.dims>` defines which 2D or 3D region is currently being viewed.
+2. Slicing: [`Layer._slice_dims`](https://github.com/napari/napari/blob/b3a8dd22895c913d8183735f52b9d1d71c963d7f/napari/layers/base/base.py#L1184) loads the corresponding 2D or 3D region of the layer's ND data into RAM.
+3. Drawing: [`VispyBaseLayer._on_data_change`](https://github.com/napari/napari/blob/b3a8dd22895c913d8183735f52b9d1d71c963d7f/napari/_vispy/layers/base.py#L126) pushes the 2D or 3D sliced data from RAM to VRAM to be drawn on screen.
+
+But as the details of this document reveal, rendering in napari is, in fact, very complicated.
+
+Consider some of the more important reasons for this.
+
+- Multiple layers can have different extents with different transforms.
+- Different layer types (e.g. Images vs. Points) handle slicing differently.
+- Layer data can be large or slow to load into RAM.
+- Sliced data in VRAM may exceed the maximum texture size.
```

Comment:
Can we expand just with a few more words here for newbies like me sorry? I would love to see something like "maximum texture size of the graphics card" or "maximum texture size of vispy" or idk, as you can tell I don't know who's maximum texture size this is 😆

Reply:
Incoming.